### PR TITLE
Add ForwardWord and BackwardWord  in console

### PR DESCRIPTION
**Pull request by kagikn**

### CODE REVIEW

File: source/core/Console.cs [hunk 1]
```diff
@@ -535,6 +533,18 @@ internal void OnKeyDown(object sender, KeyEventArgs e)
 						return;
 				}
 			}
+			else if (e.Alt)
+			{
+				switch (e.KeyCode)
+				{
+					case Keys.B:
```
[the remark this answers is not in the document]
Comment (kagikn) [reply source/core/Console.cs]: I think This doesn't collide, I haven't tested though.
But if you tried shortcuts in shells like bash a lot of times, you wouldn't prefer Ctrl+Left arrow over Alt+F to forward word. 
In fact, the former requires you to move your finger more than the latter (and Ctrl+an arrow key is bound to a different input in the shells by default).
Anyway, I'll wait for crosire's desition.

[the remark this answers is not in the document]
Comment (crosire) [reply source/core/Console.cs]: Why not both?

[the remark this answers is not in the document]
Comment (JohnnyCrazy) [reply source/core/Console.cs]: Or even configurable? :P

@kagikn 
TBH I've never seen ALT + F as navigator, that's why I'm kinda surprised. I just tested ZSH, CMD, PowerShell and VSCode/Atom, and they all use CTRL + Arrow keys for me. I think this heavily depeends on your keyboard-layout/language, doesn't it?

[the remark this answers is not in the document]
Comment (kagikn) [reply source/core/Console.cs]: @JohnnyCrazy You're right about CTRL + Arrow keys (In vim, the arrow keys are considered bad habits though). And I realized Alt + F in zsh bahaves differently (moves the caret to the start of the next word) from Emacs and shells like bash and fish.

**Comment by kagikn** [comment]
Added shortcuts of Ctrl + Left or Right arrow keys, for now.

**Pull request merged**